### ISSUE TITLE
Is this stable enough to use in production as a basic distributed cache?

### ISSUE THREAD

**Issue by jonbonazza** [hazelcast/hazelcast-go-client#321]
We only intend to use hazelcast maps. Is it safe to use in production?

**Comment by dbrimley** [comment]
Hi Jon,

TL;DR : We consider the client to be usable in production from a stability perspective, but beware that the API may change from version to version for a time.

The GO library is put through the same testing procedures and 48 hour soak tests as all our other clients, so in regards to stability we consider it safe to use in production.  It is still a relatively new client however so of course we do expect to hear of issues as more people use it.

The versioning will remain at 0.x for a period of time whilst we stabilize the API and bring it to feature completeness.  Therefore we do reserve the right to make breaking changes to the API from version to version, although we always try to keep these disruptions to a minimum.

When we consider the client to be feature complete and API stable we will move the version to align with the IMDG releases.  So it will become for example 3.12 or 3.11.

Hope this helps, and we look forward to hearing about your experiences.

**Comment by dbrimley** [comment]
@jonbonazza Be keen to hear of your experiences with the client going forward and if you made it into production.  Reach out to us if you need help.

In the meantime I'll close this issue.
